pkg/application/v2app.go

```diff
@@ -167,6 +166,7 @@ func NewV2Application(app *v2.Application, extension imagev1.ImageExtension) (In
 		globalCmds:          extension.Launch.Cmds,
 		appLaunchCmdsMap:    map[string][]string{},
 		appRootMap:          map[string]string{},
+		appEnvMap:           map[string]map[string]string{},
```

Comment:
should we init the appEnvMap here to avoid the map objects with a value of nil being used? same with the others